frontend/src/Editor/Inspector/EventManager.jsx

```diff
@@ -1,11 +1,13 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { ActionTypes } from '../ActionTypes';
 import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
 import Popover from 'react-bootstrap/Popover';
-import SelectSearch, { fuzzySearch } from 'react-select-search';
 import { CodeHinter } from '../CodeBuilder/CodeHinter';
 import { GotoApp } from './ActionConfigurationPanels/GotoApp';
+import Select from '../../_ui/Select';
+import defaultStyles from '../../_ui/Select/styles';
```

Comment:
For better readability, we can import using `@`

```suggestion
import Select from '@/_ui/Select';
import defaultStyles from '@/_ui/Select/styles';
```